example/main.dart

```diff
@@ -116,6 +116,11 @@ class _MyHomePageState extends State<MyHomePage> {
           <h1>Second header</h1>
           <h1>Third header</h1>
           <div>Fourth div</div>
+          <details>
+            <Summary open>Details summary</summary>
```

Comment:
```suggestion
            <summary open>Details summary</summary>
```
Just a quick typo fix.